### ISSUE TITLE
Huge difference to regular darknet output

### ISSUE THREAD

**Issue by ItsMeTheBee** [guichristmann/edge-tpu-tiny-yolo#12]
Hey there!

After compiling my custom tiny yolo v3 network for the edgetpu I´m able to run my network but the output is insanely different.
On my custom network there is only one class, anchors and input size remain the same so i don´t know where these issues might be coming from.

Edge Tpu output:
![result](https://user-images.githubusercontent.com/37577432/89284580-48c74200-d64f-11ea-96a9-fad2d54ba0b5.png)

Desired output
![predictions](https://user-images.githubusercontent.com/37577432/89284633-60062f80-d64f-11ea-9620-686ecc9e3778.jpg)

Do you have any why this is happening and how to fix this?

**Comment by daynauth** [comment]
I've been having the same issue too. I've seen a notable loss in accuracy with the conversion from darknet to keras and this may have been propagated onto the edge tpu model.

**Comment by ownbee** [comment]
I am experiencing the same kind of issue (worse output after conversion). I don't know if it's the inference.py script or the converted model. I've also noticed that the boxes sometimes have negative coordinates.

**Comment by Rariusz** [comment]
I have the same problem, any solution?

**Comment by YsYusaito** [comment]
Hi @ItsMeTheBee @Rariusz 
I corrected mistakes of utils.py.(Issue #19)
I hope it helps you solve your problem.

Plus, please confirm that 「anchor.txt」 is loaded properly. (I think your bounding boxes are too small)

**Comment by stan-guer** [comment]
Maybe it has something to do with the missing representative dataset for int8 calibration? This repo for some reason just generates random data.
Compare:
https://github.com/guichristmann/edge-tpu-tiny-yolo/blob/master/keras_to_tflite_quant.py#L21
to
https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/master/convert_tflite.py#L17
 
Unfortunately, I couldn't get this repository to work yet, so I can't confirm if this is a fix.

**Comment by lukqw** [comment]
Because my inference also yields very bad results, I ran this through the [quantization debugger](https://www.tensorflow.org/lite/performance/quantization_debugger).

Results include two very high RMSE / scale values, for functional_1/activation_5/Relu and functional_1/activation_7/Relu, which probably lead to this bad accuracy.

Full results can be found in the attached csv

@guichristmann can you please shine light on if the example gif of joker provided was performed with the edgetpu compiled version of the model? Additionally, could you please tell us what dataset you used as a representative dataset when converting to tflite?

[debugger_results.csv](https://github.com/guichristmann/edge-tpu-tiny-yolo/files/7532254/debugger_results.csv)
